java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -2318,7 +2322,7 @@ protected void calculateTrackSegmentAngle() {
                 }
             }
         }
-    }   // calculateTrackSegmentAngle
```

Comment:
These comments are useful when scrolling through long methods. Please don’t remove them.

Reply:
I got that, but sorry they are inconsistently applied, and then just in a few LE classes so I clean them up.
Hint: an IDE will reveal the same info when Breadcrumbs is turned on.

Reply:
@silverailscolo I use vi for most of my editing.

I add end of method markers  when I need them to keep track of where things are.  Some methods are easy to follow without them, others are not.

Reply:
I left them in place (put them back in) when a method doesn't fit on a medium size screen = length of 60 lines. Anything shorter makes the code less readable to at least 1 user as the comment hides the ``}`` flowing back to the left margin. Fine while working on code, use whatever works.